### PR TITLE
Add delete registry

### PR DESCRIPTION
**Pull request by maridob**
#99 Add Delete in swagger

### CODE REVIEW

File: components/api_docs/voice-registry-system-v1.json [hunk 1]
```diff
@@ -196,6 +196,52 @@
...
+            "description": "Unique identifier for the record",
+            "required": true,
+            "schema": {
+              "type": "string",
+              "format": "uuid",
+              "example": "ef7d274e-d5e5-11e7-9296-cec278b6b50a"
+            }
+          }
+        ],
+        "responses": {
```
Comment (rogerkibbe) [inline components/api_docs/voice-registry-system-v1.json:229]: Needs a 200 e.g. success?

File: components/api_docs/voice-registry-system-v1.json [hunk 2]
```diff
@@ -196,6 +196,52 @@
...
+            "name": "api_key",
+            "in": "header",
+            "required": false,
+            "schema": {
+              "type": "string",
+              "example": "SJHDSJHJN-JSHDJHSD-AISK12"
+            }
+          },
+          {
+            "in": "path",
```
Comment (rogerkibbe) [inline components/api_docs/voice-registry-system-v1.json:218]: Should our API use UUID's or URL's - worth a discussion

**Review by rogerkibbe — COMMENTED** [review]
Generally looks good, a couple of thoughts
- Define a 200 success 
- Should the API use a UUID or a URL for the record identifier - worth a discussin

**Review by maridob — COMMENTED** [review]
Base on the VRS meeting conversation today. This is approved.

**Pull request merged**